azurefuncs/abc.py

```diff
@@ -0,0 +1,58 @@
+import abc
```

Comment:
Nit: could we rename this file something other than abc?

Reply:
`abc` is a standard Python abbreviation for Abstract Base Class and it's a pretty standard naming. Do you have a better name in mind?

Reply:
Looks like there is a standard library abc. Just wanted to make sure our naming is not confusing.

Reply:
Yeah, that could be the case, but I'd prefer people not to import this module from `azure.functions` at all.  I guess we should just rename it to `_abc.py`, so that people will be forced to use `azure.functions` directly (and not `azure.functions.abc`.